lib/config/index.js

```diff
@@ -0,0 +1,46 @@
+module.exports = getConfig
+
+var statSync = require('fs').statSync
+var path = require('path')
+
+var defaultsDeep = require('lodash').defaultsDeep
+var series = require('async').series
+
+var accountConfig = require('./account')
+var assureFolders = require('./assure-folders')
+var couchDbConfig = require('./db/couchdb')
+var getDefaults = require('./defaults')
+var pouchDbConfig = require('./db/pouchdb')
+var storeConfig = require('./store')
+
+function getConfig (server, config, callback) {
+  defaultsDeep(config, getDefaults())
+
+  var adapter = config.PouchDB.preferredAdapters[0]
```

Comment:
The idea is that `@hoodie/server` does not have `pouchdb` as dependency, hence the `.PouchDB` option. Does that make sense? 

Note that you’ll still be able to require `hoodie` as package and it will continue to return a hapi plugin, but if you only need APIs and need more control, you can now directly require the "lower level" `@hoodie/server`. 

> it conflicts with this line in the readme at present: https://github.com/hoodiehq/hoodie-server/pull/500/files#diff-04c6e90faac2675aa89e2176d2eec7d8R59

Good catch, I’ll fix it

> Providing the user does what this file did (https://github.com/hoodiehq/hoodie/blob/82897fd9da994e88f4a0aae30e7c6de02f04dad1/server/config/db/factory.js), then it works? (I'm about to test)

We actually don’t need this factory workaround any longer, because PouchDB’s `prefix` option now applies to the http adapter as well, so we can get rid of it :)